server/subscribe.go

```diff
@@ -114,6 +114,61 @@ func (p *Plugin) getUserID() string {
 	return p.getConfig().botUserID
 }
 
+func (p *Plugin) matchesSubsciptionFilters(wh *webhook, filters SubscriptionFilters) bool {
```

Comment:
_begging_ for unit tests now :) (separate PR)

Reply:
Thanks for extracting this into its own function :+1: